pysces/PyscesParScan.py

```diff
@@ -31,7 +31,8 @@
 import numpy as np
 from pysces.PyscesUtils import TimerBox
 from pysces.PyscesScan import Scanner
-import sys, os, pickle
+import sys, os
+import dill as pickle
```

Comment:
Note to self. @bgoli dill looks cool, see if you can standardise other PySCeS pickle using functions on it.